odoo/tools/mail.py

```diff
@@ -381,7 +381,7 @@ def create_link(url, label):
     return f'<a href="{url}" target="_blank" rel="noreferrer noopener">{label}</a>'
 
 
-def html2plaintext(html, body_id=None, encoding='utf-8'):
+def html2plaintext(html, body_id=None, encoding='utf-8', addUrls=True):
```

Comment:
It is more about references than links, as it contains links and images, so maybe
```suggestion
def html2plaintext(html, body_id=None, encoding='utf-8', include_references=True):
```